server.py

```diff
@@ -155,16 +147,45 @@ def get_responses(tx_id=None, invalid=None):
                      error=e)
 
 
-def json_serial(obj):
-    """JSON serializer for objects not serializable by default json code"""
-    if isinstance(obj, datetime):
-        return obj.isoformat()
-    raise TypeError("Type not serializable")
+def get_size_and_hash(tx_id=None, invalid=None):
```

Comment:
Maybe this should be called `get_submission_metadata`? What would the name be if we add another value?
The name also implies that the function is doing 2 things